tox.ini

```diff
@@ -15,7 +15,6 @@ commands =
       -W error::ResourceWarning \
       -W error::DeprecationWarning \
       -W error::PendingDeprecationWarning \
-      -W ignore:datetime.datetime.utcfromtimestamp:DeprecationWarning \
```

Comment:
there's also a warning from dateutil that needs suppressing, so unfortunately I need to restore this